implementations/rust/ockam/ockam_command/tests/bats/projects_orchestrator.bats

```diff
@@ -59,6 +60,7 @@ teardown() {
   OCKAM_HOME=$NON_ENROLLED_OCKAM_HOME
 
   # Green' identity was added by enroller
+  run_success "$OCKAM" project import --project-file $ENROLLED_OCKAM_HOME/project.json
```

Comment:
Just checking if IIUC,  instead of 
[ockam project enroll --identity $identity --project-path $project_json_path](https://github.com/build-trust/ockam/blob/9ac68511f23e80834e6ce70a440eab676f5b060f/implementations/rust/ockam/ockam_command/tests/bats/projects_orchestrator.bats#L60)

we now have to do
[  ockam project import --project-file $project_json_path
  ockam project enroll --identity $identity](https://github.com/build-trust/ockam/blob/75b78164dfbe6630b8fe74715b0bcf4714d88a9f/implementations/rust/ockam/ockam_command/tests/bats/projects_orchestrator.bats#L79)

there is likely many places where that is needed?

Reply:
I think it was only needed in that place because in other places we do the copy right at the beginning of the test.